.github/workflows/shared-build-and-test.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
           path: target/site/jacoco/*
 
       - name: Vulnerability Scan
-        uses: IABTechLab/uid2-shared-actions/actions/vulnerability_scan_filesystem@v2.4.0
+        uses: IABTechLab/uid2-shared-actions/actions/vulnerability_scan_filesystem@kcc-UID2-2861-fix-github-warnings
```

Comment:
Will replace this and all the following `kcc-UID2-2861-fix-github-warnings` to `v2` once approved